src/core/Song.cpp

```diff
@@ -1239,7 +1233,6 @@ bool Song::saveProjectFile(const QString & filename, bool withResources)
 
 	saveState( dataFile, dataFile.content() );
 
-	m_globalAutomationTrack->saveState( dataFile, dataFile.content() );
```

Comment:
Shouldn't the member `m_globalAutomationTrack` be fully removed as well?

Reply:
Yes, but in another PR.

Reply:
To elaborate, That is the step 4 in #403, which should be done after #5230(step 3).

Reply:
Ok, thanks for the heads up! :+1: